### PR TITLE
Make http_server example client actually log

### PR DESCRIPTION
**Pull request by tiziano88**
It should help debugging the issue described in https://github.com/project-oak/oak/pull/1461#discussion_r486658233

# Checklist

- [ ] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [ ] I have written tests that cover the code changes.
  - [ ] I have checked that these tests are run by
        [Cloudbuild](/cloudbuild.yaml)
  - [ ] I have updated [documentation](/docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

File: examples/http_server/client/src/main.rs [hunk 1]
```diff
@@ -68,6 +69,12 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .await
         .expect("Error while awaiting response");
 
-    log::info!("Got response: {:?}", resp);
+    log::info!("response: {:?}", resp);
```
Comment (tiziano88) [inline examples/http_server/client/src/main.rs:72]: BTW it seems that the response headers (e.g. content-type) are not there when this is logged, @rbehjati , not sure whether they are lost along the way, or I am doing something wrong when logging

Comment (rbehjati) [reply examples/http_server/client/src/main.rs:72]: Nope! My bad. The headers are not set in the response! I can fix it with #1458.

Comment (tiziano88) [reply examples/http_server/client/src/main.rs:72]: SG, thanks! I think this PR can go ahead without that anyways (and hopefully will help to check that issue is fixed, once you have a PR for it)

File: examples/http_server/client/src/main.rs [hunk 2]
```diff
@@ -68,6 +69,12 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
...
+    log::info!(
```
Comment (rbehjati) [inline examples/http_server/client/src/main.rs:73]: Does not the previous line already print the body of the response too?

**Review by rbehjati — APPROVED** [review]

File: examples/http_server/client/src/main.rs (hunk 2 above)
Comment (tiziano88) [reply examples/http_server/client/src/main.rs:73]: No, it just prints `Body(stream)` or something like that.

**Pull request merged**